android/build.gradle

```diff
@@ -25,18 +25,29 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 33
+    namespace "com.mparticle.mparticle_flutter_sdk"
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'
     }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_17
+    }
```

Comment:
When I pulled this down and tried to run the example app in Android, I got the following error:

```
FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':mparticle_flutter_sdk:compileDebugJavaWithJavac'.
> error: invalid source release: 17
```

Changing the 3 instances of `JavaVersion.VERSION_17` to `JavaVersion.VERSION_1_8`, or removing lines 34 through 41 resolved this for me and I was able to run the example app successfully.

Reply:
Thanks for the feedback.

I have downgraded them all down to 1.8.